docs/src/real_world.md

```diff
@@ -33,8 +33,8 @@ save(datadir("sim", "barkley", "astonishing_results.jld2"), data)
 ```
 
 ## Making your project a usable module
-For some projects, it is often the case that some packages and files from the source folder are loaded at the beginning of _every file of the project_.
-For example, I have a project that I know that for _any_ script I will write, the first five lines will be:
+By default DrWatson.jl's projects are **not** Julia packages since that leads to recompilation of the project every time a source file is changed which can take a substantial amount of time. However, in some cases it can be beneficial nonetheless to turn the project into a usable module, e.g. if you want to use [`Revise.jl`](https://github.com/timholy/Revise.jl) for development. There are also some projects, where it is the case that some packages and files from the source folder are loaded at the beginning of _every file of the project_.
+Let's consider the example, where I have a project that I know that for _any_ script I will write, the first five lines will be:
```

Comment:
I would revert this back to what it was. Let's keep the discussion here simple. Besides, we don't really discuss packages in this section. The recompilation problem of issue #397 isn't as clear as we thought, so, as you correctly pointed out, this recompilation doesn't happen always, only some particular, difficult to reproduce cases. Hence, we shouldn't say too much abbout this here.